### ISSUE TITLE
Edit Screen Not Loading

### ISSUE THREAD

**Issue by velkymx** [ProcessMaker/spark-screen-builder#200]
On develop, click on edit of any of the screens and they do not load.

https://develop.spark.qa.processmaker.net/processes/screen-builder/1/edit

<img width="1671" alt="Screenshot 2019-05-22 14 20 30" src="https://user-images.githubusercontent.com/535300/58210075-34429800-7c9d-11e9-8436-3fa71324ae42.png">

Spark Screen Export

[get_user_information.spark.txt](https://github.com/ProcessMaker/spark-screen-builder/files/3209371/get_user_information.spark.txt)

**Comment by chipit24** [comment]
@velkymx I can't seem to reproduce this on the latest develop branch (in spark) with the latest spark-screen-builder develop branch npm link'ed in. The editor loads just fine, and import works as expected as well, with the imported form also loading without issue.

I noticed you're running the older screen builder ("Drag element here" is still present), and when I tested with that, that issue was present. So the solution might be to publish the latest screen builder and pull it into spark.

One thing I noticed as I was going back-and-forth testing on develop with and without the latest screen builder symlinked in, is that the screen builder seemed to be broken as you described, in both cases, when I saw the Socket.io error in the console; that error seems unrelated to screen builder but that's the only change I noticed when I tested the 2nd time.

Here's a video of it working on latest spark develop with latest spark-screen-builder pulled in:
https://www.dropbox.com/s/xl36xwug58wt20j/import_working.mov?dl=0
